docs/docs/ecosystem/build-on-composable-ecosystem-development.md

```diff
@@ -4,133 +4,31 @@
-Instrumental will automatically deploy the funds to the highest yield-earning platform and pool currently available, 
-regardless of the chain or layer it is on.
+- [Angular Finance](https://www.angular.finance/)
 
-### Whirlpool Cash:
-
-[Whirlpool Cash](https://www.whirlpoolcash.com/) is a native, 
-privacy-preserving protocol that works by mixing transactions into a collective, anonymized pool. 
-In addition, Whirlpool uses yield farms to generate returns for 
-those who deposit assets that are part of Composable’s ecosystem, which are later paid back to contributors. 
-This compensates for any costs incurred through transaction fees 
-and inverts the traditional view of privacy in DeFi as a cost-driver.
-
-To see an overview of the decentralized applications and services hosted within the Composable ecosystem, explore
-[ this blog by Karel Kubat and 0xbrainjar](https://medium.com/composable-finance/the-primitives-needed-to-power-a-complete-defi-ecosystem-6e7d5cfdc525).
-
-## Composable Labs Roadmap
-
-### Phase 1
-
-Composable Labs will develop and run in-house and incubated projects. 
-These include Substrate primitives such as Angular Finance, Ensemble, and Whirlpool Cash. 
-During each phase, Composable Labs will continue researching innovative solutions for DeFi products and services.
-
-### Phase 2
-
-Composable Labs will dedicate members of its talent pool to project-specific needs and enable developers to use 
-the resources available at Composable to build faster, more efficiently, and scale at will. 
-Angular, for example, already has a dedicated team of Composable Labs engineers to aid in the project’s development.
-
-### Phase 3
-
-During this phase, Composable will be entirely up and running, and the pallets on Picasso. 
-Composable Labs will heavily explore the nested parachain approach to project deployment on the Composable parachain. 
-This means that each project will become a nested parachain.
-
-## Composable Research
-
-Composable Labs is always working towards advancing innovative ideas, 
-identifying market trends, and discovering untapped potential. 
-This is not only the case for in-house projects – Composable Labs works closely with incubated project teams to identify 
-problems, shortcomings, and opportunities for new functionality or increased efficiency. 
-
-Composable Labs have recently researched and developed Instrumental Finance’s pallet on Picasso. 
-This benefits mainnet and EVM ecosystem users looking to farm yield on Picasso and DotSama. 
-As Picasso-based pallets start offering liquidity rush events (LREs) and other strategies, 
-Ethereum and other EVM communities can participate in these fresh APY opportunities without the hassle of installing a 
-Polkadot.js wallet. 
-
-Composable Labs works in tandem with our in-house lead cryptography researcher, 
-[@Cryptoni0x](https://twitter.com/Cryptoni0x), who plays a crucial role in researching cryptographic primitives and the 
```

Comment:
keep this paragraph and the next paragraph, delete cryptonymph in there, but yeah composable research can be kept in